sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -70,7 +70,12 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils {
   private val sqlContext = _sqlContext
 
   test("UDTF") {
-    sql(s"ADD JAR ${TestHive.getHiveFile("TestUDTF.jar").getCanonicalPath()}")
+    val jarPath = TestHive.getHiveFile("TestUDTF.jar").getCanonicalPath
+
+    // SPARK-11595 Fixes ADD JAR when input path contains URL scheme
+    val jarURL = s"file://$jarPath"
```

Comment:
Is it possible to remove an added jar? I'm afraid if we add the other test case, these two may interfere with each other.